client/src/common/client.ts

```diff
@@ -163,7 +163,7 @@ interface ConnectionCloseHandler {
 
 interface ConnectionOptions {
     cancellationStrategy: CancellationStrategy
-    restartTimes: number
+    maxRestartCount: number
```

Comment:
Actually more stuff. Sorry for this. For backwards compatibility this should be in the whole call sequence be optional.

Reply:
Done.